### ISSUE TITLE
several loaders support

### ISSUE THREAD

**Issue by dreyks** [prateekbh/babel-esm-plugin#7]
this code
```
test: /\.coffee$/,
  use: [
    {
      loader: 'babel-loader',
      options: {
        'presets': [
          [
            '@babel/preset-env', {
            'targets': {
              'browsers': ['last 2 versions', 'safari >= 7']
            }
          }
          ]
        ]
      }
    },
    'coffee-loader'
  ]
```
fails with `Babel-loader config not found!!!` because plugin code does not expect `use` to be an array

**Comment by prateekbh** [comment]
Oh nice! I'll get to this once back  from vacation

**Comment by prateekbh** [comment]
Feel free to PR if u have a solution

**Comment by prateekbh** [comment]
should be fixed in 0.2.2
